Gesture_control.py

```diff
@@ -8,6 +8,34 @@
 import cv2
 import numpy as np
 
+
+def get_trackbars():
```

Comment:
Rename this function to create_trackbars. That's what it is doing.
Also, call this function in the code somewhere outside the while loop